vllm/model_executor/models/__init__.py

```diff
@@ -182,6 +181,15 @@ def register_model(model_arch: str, model_cls: Type[nn.Module]):
     def is_embedding_model(model_arch: str) -> bool:
         return model_arch in _EMBEDDING_MODELS
 
+    @staticmethod
+    def is_multimodal_model(model_arch: str) -> bool:
+        model_cls = ModelRegistry._try_load_model_cls(model_arch)
+
+        # Avoid circular import
+        from vllm.model_executor.models.interfaces import supports_multimodal
+
+        return supports_multimodal(model_cls)
```

Comment:
@youkaichao is there a good way to avoid initializing CUDA prematurely here? We shouldn't rely on not importing modules that initialize CUDA inside the model files.

Reply:
I can't think of anything outside of performing this check in a separate process.

Reply:
Will use a hardcoded class of multimodal models as a workaround for now